apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -3,9 +3,33 @@
     <oc-table-row v-if="$_reshareInformation" class="files-collaborators-collaborator-table-row-top">
       <oc-table-cell shrink :colspan="firstColumn ? 2 : 1"></oc-table-cell>
       <oc-table-cell colspan="2">
-        <div class="uk-text-meta uk-flex uk-flex-middle">
-          <oc-icon name="repeat" class="uk-preserve-width" />
-          <span class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</span>
+        <div class="uk-text-meta">
+          <a :id="$_resharerToggleId" :aria-label="$gettext('Show resharer details')" class="uk-flex uk-flex-middle">
```

Comment:
Shouldn't it be displayed only if there are any resharers?

Reply:
the check is done higher above on the table row

Reply:
I see now - overlooked it before. Nice, thanks!